extensions/wakatime/src/today-summary.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSummary } from "./hooks";
+
+import { MenuBarExtra } from "@raycast/api";
+
+export default function Command() {
+  const summary = useSummary();
+
+  return (
+    <MenuBarExtra
+      icon="../assets/icon.png"
+      title={`${summary.data
+        ?.map(
+          ([key, range]) =>
+            key === "Today" &&
+            (range.cummulative_total.seconds === 0 ? "Are we coding today?" : `Today: ${range.cummulative_total.text}`)
```

Comment:
I think this should be an optional preference. Something that asks if they want to reminded the person hasn't recorded any time today.

Reply:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/30423976/182009214-5155c205-5da9-4bbf-812f-eddd933ca5ee.png">

You can make it be empty if you do not want to have any asks.